runtime/rasdump/javadump.cpp

```diff
@@ -2021,7 +2021,7 @@ JavaCoreDumpWriter::writeHookInfo(struct OMRHookInfo4Dump *hookInfo)
 	_OutputStream.writeCharacters(timeStamp);
 	_OutputStream.writeInteger64(hookInfo->startTime % 1000, ".%03llu");
 	_OutputStream.writeCharacters("\n");
-	_OutputStream.writeCharacters("4HKDURATION           DurationMs: ");
+	_OutputStream.writeCharacters("4HKDURATION           Durationμs: ");
```

Comment:
I don't think we want non-ASCII content in javacore files.

Reply:
@cedrichansen please, fix. we commonly use plain 'u' instead.

Reply:
will do!